### PR TITLE
Change min req of python-sdk to python 3.7 from python 3.8 

### PR DESCRIPTION
**Pull request by pruthvidhodda**
# Description

Made changes to support using python-sdk with Python 3.7

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #82 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -10,13 +10,13 @@ classifiers =
     License :: OSI Approved :: MIT License
     Operating System :: OS Independent
     Programming Language :: Python
-    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.7
```
Comment (youngbupark) [inline setup.cfg:13]: Add python 3.8 support as well.

```
Programming Language :: Python :: 3.8
```

File: ext/flask_dapr/setup.cfg [hunk 2]
```diff
@@ -10,13 +10,13 @@ classifiers =
     License :: OSI Approved :: MIT License
     Operating System :: OS Independent
     Programming Language :: Python
-    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.7
```
Comment (youngbupark) [inline ext/flask_dapr/setup.cfg:13]: please add 3.8 support
```
Programming Language :: Python :: 3.8
```

File: tests/actor/fake_actor_classes.py [hunk 3]
```diff
@@ -6,10 +6,78 @@
...
 from datetime import timedelta
+from typing import Optional
 
 from dapr.actor.runtime.actor import Actor
 from dapr.actor.runtime.remindable import Remindable
 from dapr.actor.actor_interface import ActorInterface, actormethod
+from dapr.clients import DaprActorClientBase
+
+
+# Fake Dapr Actor Client Base Class for testing
```
Comment (youngbupark) [inline tests/actor/fake_actor_classes.py:17]: please create `fake_client.py` and move fake client to new file.

File: tests/actor/test_actor.py [hunk 4]
```diff
@@ -18,20 +18,26 @@
...
-class ActorTests(unittest.IsolatedAsyncioTestCase):
-    async def asyncSetUp(self):
+class ActorTests(unittest.TestCase):
+    def setUp(self):
         ActorRuntime._actor_managers = {}
         ActorRuntime.set_actor_config(ActorRuntimeConfig())
         self._serializer = DefaultJSONSerializer()
-        await ActorRuntime.register_actor(FakeSimpleActor)
-        await ActorRuntime.register_actor(FakeMultiInterfacesActor)
+        _run(ActorRuntime.register_actor(FakeSimpleActor))
```
Comment (youngbupark) [inline tests/actor/test_actor.py:39]: use asyncio.run()

Comment (pruthvidhodda) [reply tests/actor/test_actor.py:40]: As discussed offline asyncio.run() can not be used more than once in the same thread and should only be used as a main entry point.

File: tests/actor/fake_client.py [hunk 5]
```diff
@@ -0,0 +1,69 @@
+
```
Comment (youngbupark) [inline tests/actor/fake_client.py:1]: License statement missing. please add the license.

**Pull request merged**